tools/libxl/libxl_internal.c

```diff
@@ -575,6 +575,24 @@ void libxl__update_domain_configuration(libxl__gc *gc,
     dst->b_info.video_memkb = src->b_info.video_memkb;
 }
 
+int libxl__is_driver_domain(libxl__gc *gc, uint32_t domid)
+{
+    const char *val;
+    int rc;
+
+    char *dom_path = libxl__xs_get_dompath(gc, domid);
+
+    if (!dom_path) {
```

Comment:
You can omit braces.

Reply:
No way. 
https://softwareengineering.stackexchange.com/questions/16528/single-statement-if-block-braces-or-no

Reply:
So why you don't put braces here:
if (rc) return 0;

Reply:
https://www.kernel.org/doc/html/v4.10/process/coding-style.html#placing-braces-and-spaces

Reply:
Because it is one line and common for libxl code:
> Error code paths with an if statement and a goto or a return on the same line are allowed.

Previous case it is not error path it is normal check and shouldn't be put as one line.